plans/main.fmf

```diff
@@ -0,0 +1,44 @@
+prepare:
+    - name: Install dependencies
+      how: install
+      copr: rhcontainerbot/podman-next
+      package:
+        - aardvark-dns
+        - bats
+        - bind-utils
+        - cargo
+        - clippy
+        - dbus-daemon
+        - dnsmasq
+        - firewalld
+        - go-md2man
+        - jq
+        - kernel-modules-core
+        - make
+        - nmap-ncat
+        - podman
+        - protobuf-c
+        - protobuf-compiler
+        - rustfmt
+    - name: Start firewalld and dnsmasq
+      how: shell
+      script: systemctl start firewalld dnsmasq
```

Comment:
Test run in a separate namespace so this doesn't do anything for the test.
The test will start the services via cli so as long as the binary is installed it should be fine.

Reply:
ack, thanks